### PR TITLE
[CI] Add php 7.3 to Travis

### PR DESCRIPTION
**Pull request by andreybolonin**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| License       | MIT

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
That's all red: it looks like some extensions are not ready for 7.3 yet.

Status: needs work

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@ matrix:
           env: deps=high
         - php: 7.2
           env: deps=low
+        - php: 7.3
+          env: deps=low
```
Comment (stof) [inline .travis.yml]: we should have only one `deps=low` job IMO (and as 7.3 is not stable yet, using it for the low deps is weird as they might trigger PHP deprecations or be incompatible btw).

Please remove the `deps=` config entirely here.

**Comment by stof** [comment]
I checked what happens. Our CI build enables the `memcached` extension which is meant to be preinstalled by Travis. But their PHP 7.3 version does not preinstalls it, because releases are not compatible yet (https://github.com/php-memcached-dev/php-memcached/issues/385).

@nicolas-grekas would it be easy to skip the loading of memcached.so on PHP 7.3 in the meantime ?

**Comment by stof** [comment]
The pecl installation of extensions should be updated to use the latest version (old versions have no chance to be compatible with PHP 7.3)

**Comment by nicolas-grekas** [comment]
Closing as discussed. There is more work to do to have it green.

**Pull request closed without merging**